server/settings.py

```diff
@@ -195,14 +201,32 @@ def env(variable, fallback_value=None):
 <p>{{ item.description_text }}{% if item.get('links') %} URL: {{ item.links | join(' ') }}{% endif %}</p>
 {% endif %}
 {% if item.contacts %}{% for contact in item.contacts %}
-<p>{{ contact.honorific }} {{ contact.first_name }} {{ contact.last_name }}
-{% if contact.contact_email %} - {{ contact.contact_email|join(' - ') }}
-{% endif %}
-{% if contact.contact_phone %} - {{ contact.contact_phone|selectattr('public')|join(' - ', attribute='number') }}
-{% endif %}</p>
+<p>{{ contact.honorific }} {{ contact.first_name }} {{ contact.last_name }}{% if contact.contact_email %} - {{ contact.contact_email|join(' - ') }}{% endif %}{% if contact.contact_phone %} - {{ contact.contact_phone|selectattr('public')|join(' - ', attribute='number') }}{% endif %}</p>
 {% endfor %}{% endif %}
 {% if item.event and item.event.location %}
 <p>{{ item.event.location|join(', ', attribute='name') }}</p>
 {% endif %}
 {% endfor %}
 '''
+
+# planning expire items
```

Comment:
I think you don't have to add these, it's done by [planning module](https://github.com/superdesk/superdesk-planning/blob/master/server/planning/__init__.py#L168-L210)

Reply:
I see, thanks.